spec/classes/letsencrypt_scripts_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'letsencrypt::scripts' do
+  on_supported_os.each do |os, os_facts|
+    let :facts do
+      os_facts
+    end
+
+    context "on #{os}" do
+      context 'with default values' do
+        it { is_expected.to compile }
+
+        it 'contains the domain validation script' do
+          is_expected.to contain_file('/usr/local/sbin/letsencrypt-domain-validation').
+            with_ensure('file').
+            with_owner('root').
+            with_group('root').
+            with_mode('0500').
+            with_source('puppet:///modules/letsencrypt/domain-validation.sh')
```

Comment:
This will no longer pass. It should be `.with_content()` now that you use `content =>` (which I do like btw)

Reply:
What regex/string should I check for?